### ISSUE TITLE
Add the ability to disable mouse control

### ISSUE THREAD

**Issue by HotLazy** [moonlight-stream/moonlight-qt#780]
Super app!  Add the ability to disable mouse control AT ALL (possibly by a combination of buttons on the gamepad).  Because  when playing intensively through the on-screen gamepad, you sometimes miss and miss the button and thereby press the screen and the mouse is activated.  I would be very grateful if you add the ability to disable.

**Comment by herroc1** [comment]
Yes, this is would be the most useful enhancement for my experience.

**Comment by baxda19** [comment]
Just noting for others who might find this in a google search:

**On the PC client:
Ctrl+Alt+Shift+Z - Toggle mouse and keyboard capture**

I believe the feature request might be referring to the android client anyway. 
But best to note the PC client does have this feature implemented.
